vlib/x/templating/dtm/README.md

```diff
@@ -0,0 +1,193 @@
+# dtm - Dynamic Template Manager
+
+A simple template manager integrated into the V project, designed to combine the power of V
+templates with Vweb, without the need to recompile the application with every change.
+
+**_This module is in the experimental phase, it lacks some features and may contain bugs_**
+
+## Quick Start
+
+Using the dynamic template manager ( named '**dtm**' in this readme) is relatively straightforward.
+You just need to create an instance and initialize it. Then, call the '**serve_dynamic_template**'
+function in the code that manages your web pages. 
+
+Before starting, at the root directory of your vweb project, you need to create a '**templates**'
+folder as well as a '**vcache**' folder. The absence of these directories at the root of your
+project will prevent you from using the dtm.
+
+Next, you must add your HTML file into the folder you previously created. ***Be aware that if the
+HTML templates are not placed in the correct directory, the DTM will return an error message
+indicating that it cannot find the template!!***
+
+A minimal quick start example:
+
+```
```

Comment:
End these 3 backslashes with `v`. This will make sure that `v check-md .` will try to compile the example, and thus prevent silent regressions, when something in dtm or vweb changes.

Reply:
It will also make github syntax highlight the example code.